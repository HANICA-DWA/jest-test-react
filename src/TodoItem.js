import React from 'react'

const TodoItem = (props) => (
  <div>
    <span className="title">{props.todo.name}</span> - <span>{props.todo.id}</span>
  </div>
)

export default TodoItem