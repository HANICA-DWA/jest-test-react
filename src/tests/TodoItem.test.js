import React from 'react'
import TodoItem from '../TodoItem'
import { renderer, shallow } from './testhelpers'

test('TodoItem should render', () => {
  const todo = {name: 'Auto wassen', id: 1}
  const comp = renderer.create(<TodoItem todo={todo} />)
  expect(comp.toJSON()).toMatchSnapshot()
  const instance = comp.root
  expect(instance.props.todo.name).toEqual(todo.name)
})
