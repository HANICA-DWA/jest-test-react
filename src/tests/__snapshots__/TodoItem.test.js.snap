// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoItem should render 1`] = `
<div>
  <span
    className="title"
  >
    Auto wassen
  </span>
   - 
  <span>
    1
  </span>
</div>
`;
